.github/workflows/repository-scanner.yml

```diff
@@ -44,11 +52,18 @@ jobs:
   bidi-scanner:
     runs-on: ubuntu-latest
     steps:
-
       - name: Checkout the repo to Scan
-        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846  # v3.0.0
+        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
+
+      - name: Checkout the Actions repo
+        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
+        with:
+          repository: stephenfuqua/Ed-Fi-Actions
```

Comment:
Temporarily checking out from my repository instead of the real one. I'm sure I've seen an article describing an environment variable that you can use here, but I can't find it again.

Reply:
Same as line 33
```suggestion
         repository: Ed-Fi-Alliance-OSS/Ed-Fi-Actions
```

Reply:
Done